### PR TITLE
Rename some functions and members

### PR DESCRIPTION
**Pull request by tahg**
As usual, the naming changes affected quite a few files :( And I tried to improve some matching of files I was hitting anyways.

### CODE REVIEW

File: LEGO1/mxomni.cpp [hunk 1]
```diff
@@ -278,18 +278,23 @@ MxResult MxOmni::Create(MxOmniCreateParam& p)
 // OFFSET: LEGO1 0x100afe90
 void MxOmni::Destroy()
 {
-	// FIXME: Stub
+	{
```
Comment (ItzSwirlz) [inline LEGO1/mxomni.cpp:281]: fix indentation

File: LEGO1/legoinputmanager.cpp [hunk 2]
```diff
@@ -51,9 +51,9 @@ void LegoInputManager::Destroy()
 {
 	ReleaseDX();
 
-	if (m_unk0x5c)
-		delete m_unk0x5c;
-	m_unk0x5c = NULL;
+	if (m_eventQueue)
```
Comment (ItzSwirlz) [inline LEGO1/legoinputmanager.cpp:54]: maybe put brackets here

File: LEGO1/legoworld.cpp [hunk 3]
```diff
@@ -26,12 +34,35 @@ void LegoWorld::Stop()
 	TickleManager()->UnregisterClient(this);
 }
 
+// OFFSET: LEGO1 0x1001f5e0
+MxLong LegoWorld::Notify(MxParam& p_param)
```
Comment (ItzSwirlz) [inline LEGO1/legoworld.cpp:38]: This *may* be an MxResult though I think other :Notify functions are MxLong's, so check on that

File: LEGO1/legoentity.cpp [hunk 4]
```diff
@@ -36,35 +28,53 @@ void LegoEntity::Reset()
...
 }
 
+// OFFSET: LEGO1 0x10010650 STUB
+void LegoEntity::ResetWorldTransform(MxBool p_inVehicle)
+{
+	// TODO
+}
+
+// OFFSET: LEGO1 0x10010790 STUB
+void LegoEntity::SetWorldTransform(MxVector3& loc, MxVector3& dir, MxVector3& up)
```
Comment (foxtacles) [inline LEGO1/legoentity.cpp:38]: Missing p_ prefix

File: LEGO1/legoentity.h [hunk 5]
```diff
@@ -34,34 +35,34 @@ class LegoEntity : public MxEntity {
...
 	// OFFSET: LEGO1 0x10001090
-	virtual void VTable0x30(undefined4 p_unk50) { m_unk50 = p_unk50; } // vtable+0x30
-	virtual void VTable0x34();                                         // vtable+0x34
-	virtual void VTable0x38();                                         // vtable+0x38
-	virtual void VTable0x3c();                                         // vtable+0x3c
-	virtual void VTable0x40();                                         // vtable+0x40
-	virtual void VTable0x44();                                         // vtable+0x44
-	virtual void VTable0x48();                                         // vtable+0x48
-	virtual void VTable0x4c();                                         // vtable+0x4c
+	virtual void SetWorldSpeed(MxFloat p_speed) { m_worldSpeed = p_speed; } // vtable+0x30
```
Comment (foxtacles) [inline LEGO1/legoentity.h:45]: Preferably we give the parameter the same name as the member

File: LEGO1/mxomni.cpp (hunk 1 above)
Comment (foxtacles) [reply LEGO1/mxomni.cpp:281]: This is now governed by `clang,-format`. If the action succeeds, nothing need be changed

File: LEGO1/legoworld.cpp (hunk 3 above)
Comment (foxtacles) [reply LEGO1/legoworld.cpp:38]: `MxResult` is a typedef for `MxLong`. However, unless we have enough evidence that all Notify functions restrict themselves to returning one of the two `MxResult` constants, we should leave it as-is

File: LEGO1/legoentity.h (hunk 5 above)
Comment (tahg) [reply LEGO1/legoentity.h:45]: > Preferably we give the parameter the same name as the member
Does this apply even for code that was in the 1996 source like this was?

Comment (foxtacles) [reply LEGO1/legoentity.h:45]: Good question. Personally I'd say yes, it's fine if we introduce some minor extra conventions like this, even if that doesn't equal 100% what we see in the 1996 source. Many things were applied inconsistently in the original source, I don't think taking over inconcistency on this level is worthwhile.

**Review by foxtacles — APPROVED** [review]
Looks good to me. Merging when conflicts are solved

**Pull request merged**